conversions/include/mrpt_bridge/vrep_conversion.h

```diff
@@ -0,0 +1,26 @@
+#ifndef VREP_CONVERSION_H
```

Comment:
Why do we have an mrpt_bridge subdirectory?
Why not put the vrep_conversions directly under `include`?